### ISSUE TITLE
[FEATURE] Add Hash of App Signing Key to README

### ISSUE THREAD

**Issue by shrimprugbysnowowl** [vitorpamplona/amethyst#972]
The project includes a link for Obtainium. Obtainium includes a feature to share a downloaded apk with [AppVerifier](https://github.com/soupslurpr/appverifier) to verify the hash of the developer app signing key.

Please post the hash of your app signing key in the README and/or submit it to be included in the AppVerifier database. (I think this happens via their Matrix Room? I'm happy to ask on your behalf once it's posted somewhere.)

**Comment by shrimprugbysnowowl** [comment]
> Based on the downloaded amethyst-fdroid-universal-v0.92.1.apk, the hash appears to be: C2:D0:AA:86:BC:B6:B6:20:90:56:1A:41:BB:E3:36:E9:8B:78:C2:D0:21:0A:49:8D:C8:85:F2:8E:13:48:CF:17

@vitorpamplona , your input here would be appreciated. Thanks!

**Comment by vitorpamplona** [comment]
We are using Nostr's https://zapstore.dev events to keep the hash of each of the release files since placing it on GitHub doesn't really do anything - Microsoft can change both the APK and the readme file at the same time to match.

**Comment by shrimprugbysnowowl** [comment]
You are correct. Ideally, the hash of your app signing key would be posted somewhere else as well to mitigate that potential, such as nostr or other.

-------- Original Message --------

**Comment by vitorpamplona** [comment]
Zap store is nostr :)

**Comment by shrimprugbysnowowl** [comment]
Could you kindly direct me to the page that lists the hash of your app signing key? I can't figure out how to retrieve the hash from Zapstore. This appears to be the page for Amethyst.
https://zapstore.dev/apps/naddr1qvzqqqr7pvpzp25sgue9vq76e48czssfx4newdtxmca3ug9gj4tmw2xrhexx4pztqqdxxmmd9emxjar0wfcxzmtsd3hkucfwv9kk2arg09ehge87zcm

Similarly, Zapstore provides the following:
"For cryptographic verification of any file ensure its hashes are always hosted on a separate server/domain.

File and APK certificate hashes for the Zapstore installer APK can be found on the Zapstore Nostr profile. Open it in your favorite client and make sure you are looking at the real Zapstore profile.

npub10r8xl2njyepcw2zwv3a6dyufj4e4ajx86hz6v4ehu4gnpupxxp7stjt2p8"

https://zapstore.dev/app

-------- Original Message --------

**Comment by vitorpamplona** [comment]
You can search for records from Amethyst's key: aa9047325603dacd4f8142093567973566de3b1e20a89557b728c3be4c6a844b on relay.zapstore.dev

Here's the current set of records for each APK, release kind and app definition kind:  
[app-release set.json](https://github.com/user-attachments/files/22906306/app-release.set.json)

**Comment by shrimprugbysnowowl** [comment]
I'm still not locating the hash of your app signing key based on the info that you provided. The hash of your signing key is constant across releases whereas the hash of the apk itself would change. It's used to verify that you were actually the one who compiled the apk.
